src/Features/Units/Converter/data_types/air_pressure.ts

```diff
@@ -0,0 +1,28 @@
+import convert from "convert-units"
+
+import { ConvertFrom, DataTypeConversion } from "../conversions"
+import { UnitSystem } from "Features/Units/types"
+
+class AirPressure extends DataTypeConversion {
+  public convertTo(value: number, unitSystem: UnitSystem): number {
+    switch (unitSystem) {
+      case UnitSystem.imperial:
+        return super.convertTo(value, unitSystem)
+      case UnitSystem.mariners:
+      case UnitSystem.metric:
+        return value
+    }
+  }
+  protected convertFrom(value: number): ConvertFrom {
+    return convert(value / 1000).from("bar")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['magic numbers' are not allowed: 1000](https://app.codacy.com/app/gmri/Neracoos-1-Buoy-App/pullRequest?prid=4354739)